docs/source/data/subscriptions.mdx

```diff
@@ -197,13 +197,23 @@ httpServer.listen(PORT, () => {
 
 </ExpansionPanel>
 
+> ⚠️ **Running into an error?** If you are using the `graphql-ws` library your specified subscription protocol must be consistent across your backend, frontend and **every other tool** you use (including [Apollo Sandbox](../studio/explorer/additional-features/#subscription-support)). For more information see [Switching from `subscriptions-transport-ws`](#switching-from-subscriptions-transport-ws).
```

Comment:
"you are" -> "you're"

comma after "library"

comma after "frontend"

comma after "information"